components/Animate/AnimateInView.tsx

```diff
@@ -14,7 +14,7 @@ type AnimateInViewProps = {
 export const AnimateInView = ({
   animation = 'zoomIn',
   once = true,
-  duration = 0.5,
+  duration = 0.6,
```

Comment:
Normalize animation duration, and I got rid of some of the duration props that were passed into components that use this wrapper